plugins/org.python.pydev.ast/src/org/python/pydev/ast/refactoring/RefactoringRequest.java

```diff
@@ -102,6 +102,11 @@ public IFile getIFileResource() {
      */
     public String inputName;
 
+    /**
+     * The representation of the selected name
+     */
+    public String rep;
```

Comment:
Please rename this to `activationToken` and note in the comments that when this is set to a non-empty value the initialName is the `qualifier`.